include/NeoLocalPlanner.h

```diff
@@ -82,6 +82,7 @@ class NeoLocalPlanner : public nav_core::BaseLocalPlanner {
 
 	ros::Subscriber m_odom_sub;
 	ros::Publisher m_local_plan_pub;
+	ros::Publisher m_target_pose_pub;
```

Comment:
I would prefer to have this feature in a new pull request. Looks like a lot going here in this PR. 

Please revert and give a new PR, so it would be easy for us to review and maintain the commit history.

Reply:
I will seperate those out into seperate PRs.